.travis.yml

```diff
@@ -4,5 +4,9 @@ python:
 
 # command to install dependencies
 install: pip install -r requirements.txt
+# enforce some code style standards
+before_script: 
+  - pip install flake8 --use-mirrors
+  - flake8 --ignore=E731,E226 .
```

Comment:
@pepol nah, this is quite far from people-pleasing, I'd rather see it as saving my own time in the future when I'd have to go through why is this enforced at all.